src/modules/75-cf/strings/strings.en.yaml

```
@@ -1412,11 +1413,10 @@ onboarding:
         await client.waitForInitialization();
 
         try {
-          // Log the state of the flag every 10 seconds
-          setInterval(async () => {
-            const value = await client.boolVariation(flagName, target, false);
-            console.log('Flag variation:', value);
-          }, 10000);
+        // Event happens when connection to server is established
+            client.on(Event.READY, async (flagIdentifier) => {
+              console.log(`${flagIdentifier} ready`);
+            });
```

Looks like there's some indentation issues here
```suggestion
          // Event happens when connection to server is established
          client.on(Event.READY, async (flagIdentifier) => {
            console.log(`${flagIdentifier} ready`);
          });
```

---

Thanks have fixed :)